src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,50 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    public static int splitAndSum(String input) {
+
+        if (input == null || input.isEmpty()) {
```

Comment:
```suggestion
        if (isBlank(input)) {
```
if 구문 내의 로직을 isBlank()로 분리하는 것은 의미 있을까?